eclipse-collections/src/main/java/org/eclipse/collections/impl/bimap/mutable/AbstractMutableBiMap.java

```diff
@@ -268,11 +268,7 @@ public V forcePut(K key, V value)
     @Override
     public void putAll(Map<? extends K, ? extends V> map)
     {
-        Set<? extends Map.Entry<? extends K, ? extends V>> entries = map.entrySet();
-        for (Map.Entry<? extends K, ? extends V> entry : entries)
-        {
-            this.put(entry.getKey(), entry.getValue());
-        }
+        map.forEach(this::put);
```

Comment:
This change fixes assertions in HashBiMapNoIteratorTest, which now inherits more tests/coverage from MapTestCase.